src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -225,7 +226,8 @@ void FixupParametersFromDocs (XElement api)
 		AndroidDocScraper CreateDocScraper (string dir)
 		{
 			switch (DocletType) {
-			default: return new DroidDocScraper (dir);
+			default: return new DroidDoc2Scraper (dir);
+			case JavaDocletType.DroidDoc: return new DroidDocScraper (dir);
```

Comment:
The point is that the doc scrapers are specific, in that you can't reliably use the `DroidDoc2Scraper` with `DroidDocScraper` input, right?

So if someone has:

    mono class-parse.exe --docspath=path/to/droiddoc1 foo.jar

Previously, they'll be using the `DroidDocScraper`. After this change, they'll be using the `DroidDoc2Scraper`, which won't support the previous docs, which may "break" things.

Consequently, if you can always explicitly specify the type -- and do so -- your binding won't break when we change the default scraper in the future:

    mono class-parse.exe --docspath=path/to/droiddoc1 --docstype=droiddoc foo.jar

---

That said..., the [`<ClassParse/>` MSBuild task](https://github.com/xamarin/xamarin-android/blob/a2698c2/src/Xamarin.Android.Build.Tasks/Tasks/ClassParse.cs) has no way of specifying the `ClassParse.DocletType` property, which means the MSBuild task will *always* attempt to use the "first" entry, which is `DroidDoc`, despite the fact that `ClassParse.DocumentationPaths` can contain ~anything, as it's the concatenation of `$(JavaDocPaths)`, `$(Java7DocPaths)`, `$(Java8DocPaths)`, and `$(DroidDocPaths)`.

What *presumably* would be "best" is dropping the entire concept of `ClassParse.DocletType`, and instead have [`ClassParse.CreateDocScraper()`](https://github.com/xamarin/java.interop/blob/b4e151e/src/Xamarin.Android.Tools.Bytecode/ClassPath.cs#L226-L235) "probe" the type of docs contained within `dir` so that we automatically use the appropriate scraper type.

However, we've mentioned this before and I believe that this "auto-probe" idea was considered to be difficult/impossible/flakey/inconsistent/etc., so I'm not entirely sure what's the best path forward for integration within Xamarin.Android.